### PR TITLE
Add's the Tailwind Masterclass

### PR DESCRIPTION
**Pull request by laurosilvacom**
Adds pattern to showcase mega courses. In this case, Design with Tailwind CSS Masterclass. 

The main priority is making sure when someone lands on either the main page or one of the sub-pages, they understand it's not a normal course and comes in multiple parts.

Additionally, it updates courseDependenciesData.js data (recommended resources, freshness block, topics, essential questions, quick facts). 

todos: 

- [x] update tag
- [x] update to the home page

main-page
---
sub-page

### CODE REVIEW

File: src/data/courseDependenciesData.js [hunk 1]
```diff
@@ -21,30 +21,30 @@ const courseDependencyData = (courseSlug) =>
...
-          'With utilities, everything is reusable so you rarely need to write new CSS',
+          'Every utility class is reusable so you rarely need to write new CSS',
           `Tailwind CSS makes easier to maintain a large CSS codebase `,
           'Tailwind CSS uses a mobile first breakpoint system',
           'Tailwind CSS is component-driven',
           'Tailwind CSS is single source of truth for your CSS architecture',
         ],
         essentialQuestions: [
-          'What are the best practices for building visually consistent UIs??',
+          'How do you build visually consistent UIs?',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:40]: adjusted these questions so they weren't so repetitive

File: src/data/courseDependenciesData.js [hunk 2]
```diff
@@ -21,30 +21,30 @@ const courseDependencyData = (courseSlug) =>
...
           'What are the best practices for implementing design systems? ',
-          'What are the best practices for keeping a project maintainable?',
+          'What makes a project maintainable?',
           'How much design CSS do I need to know?',
         ],
         pairWithResources: [
           {
-            id: '284646',
+            id: '227',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:47]: some id's were off

File: src/data/courseDependenciesData.js [hunk 3]
```diff
@@ -21,30 +21,30 @@ const courseDependencyData = (courseSlug) =>
...
+          'Use utilities to style elements on hover, focus, and other states',
+          'Extend Tailwind with custom utility classes',
           'Work with Tailwind CSS plugins',
           'Customize Tailwind CSS to meet the needs of your design',
           'Augment Tailwind CSS with JavaScript to make your designs interactive',
         ],
         quickFacts: [
           'Utility classes create an API on top of what is already a declarative API (CSS itself)',
-          'With utilities, everything is reusable so you rarely need to write new CSS',
+          'Every utility class is reusable so you rarely need to write new CSS',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:33]: simplify wording

File: src/data/courseDependenciesData.js [hunk 4]
```diff
@@ -21,30 +21,30 @@ const courseDependencyData = (courseSlug) =>
           'Build complex application layouts with flexbox',
           `Build accessible user interfaces following industry best practices`,
           'Removing unused CSS from production builds to maximum performance',
-          'Using responsive utility variants to build adaptive user interfaces',
-          'Using utilities to style elements on hover, focus, and other states',
-          'Extending Tailwind with custom utility classes',
+          'Use responsive utility variants to build adaptive user interfaces',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:24]: consistent verb usage

**Comment by zacjones93** [comment]
Added the tailwind course to the main page, does it make sense to include the modules in the card?

homepagewithtailwind

**Comment by MaggieAppleton** [comment]
Do we want to launch this into the jumbotron?
Have banner mockups in the dropbox: https://www.dropbox.com/sh/w4ciive7ezxi8u2/AAA6Znet9m2Lv8qdwsu_EDAga?dl=0

tailwinds_masterclass_banner1
tailwinds_masterclass_banner2

**Comment by MaggieAppleton** [comment]
If we're just keeping it as an inline card, adding the models would make sense but we'd need to style them as course links instead of just text. Adding the module images and left aligning them would fix it

**Comment by zacjones93** [comment]
What do you think about putting it up in the jumbotron? 

The thing that we were considering is that we will be having a pretty consistent schedule with a new course on Tuesday and these releases on friday

**Comment by laurosilvacom** [comment]
> Do we want to launch this into the jumbotron? @MaggieAppleton 

I'm thinking that we keep the Cloudflare course as featured for the rest of the week. We're only _upgrading_ the Tailwind Masterclass.

**Comment by zacjones93** [comment]
Updated the tailwind course card with @vojtaholik and @laurosilvacom's help

Build the portfolio you need to be a badass web developer   egghead io

**Review by vojtaholik — APPROVED** [review]
🔥

**Comment by MaggieAppleton** [comment]
Looking good! Nice job getting this built 👏

**Pull request merged**